Cotton Classing Database File Import Plugin/DataMappers/DataMapper.cs

```diff
@@ -0,0 +1,82 @@
+﻿using AgGateway.ADAPT.ApplicationDataModel.ADM;
+using AgGateway.ADAPT.ApplicationDataModel.Common;
+using AgGateway.ADAPT.ApplicationDataModel.Documents;
+using CottonClassingPlugin.PublisherDataModel;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CottonClassingPlugin.DataMappers
+{
+    public static class DataMapper
+    {
+        public static void MapData(Data myDataModel, ApplicationDataModel adm)
+        {
+            //Create an ADM Documents object
+            Documents docs = new Documents();
+            adm.Documents = docs;
+            
+            //Create a new observation object for our Documents object
+            Observations observations = new Observations();
+            var _observations = new List<Observations>();
```

Comment:
A stylistic suggestion to improve readability would be to omit the underscores from variable names within methods.  In C# this naming is generally used only for private member variables of the class.